VerilogCodingStyle.md

```diff
@@ -2533,14 +2533,16 @@ a check for `X` on the 4-state net before resolving to a 2-state variable.
 
 ### Logical vs. Bitwise
 
-***Use logical constructs for logical comparisons, bit-wise for data.***
+***Prefer logical constructs for logical comparisons, bit-wise for data.***
 
-Logical constructs (`!`, `||`, `&&`, `==`, `!=`) should be used for all
+Logical operators (`!`, `||`, `&&`, `==`, `!=`) should be used for all
 constructs that are evaluating logic (true or false) values, such as
-if clauses and ternary assignments.  Use bit-wise constructs (`~`, `|`,
-`&`, `^`) for all data constructs, even if scalar.
+if clauses and ternary assignments.  Prefer bit-wise constructs (`~`, `|`,
```

Comment:
```suggestion
if clauses and ternary assignments.  Prefer bit-wise operators (`~`, `|`,
```

(Since you changed constructs -> operators above.)

Reply:
done